gerbera-web/test/client/fixtures/clients-data.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
Looks like this clients-data.json is in two separate places?

Reply:
Using the same data for client, e2e (incl screenshots) at the moment. As tests are separate I prefer copies of the same file.